fe/src/main/java/org/apache/doris/load/loadv2/etl/EtlJobConfig.java

```diff
@@ -0,0 +1,487 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.load.loadv2.etl;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.io.Serializable;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+/** jobconfig.json file format
+ * {
+ * 	"tables": {
+ * 		10014: {
+ * 			"indexes": [{
+ * 			    "index_id": 10014,
+ * 				"columns": [{
+ * 				    "column_name": "k1",
+ * 				    "column_type": "SMALLINT",
+ * 				    "is_key": true,
+ * 				    "is_allow_null": true,
```

Comment:
Add a field `defined_expr`, and default is empty. This is for materialized view 2.0, which column may has expr definition.